package.json

```diff
@@ -54,7 +51,7 @@
     "babel-preset-env": "1.7.0",
     "babel-register": "6.26.0",
     "browser-sync": "2.26.3",
-    "bundlesize": "0.17.0",
+    "bundlesize": "^0.17.1",
```

Comment:
If this were pinned to a patch version, it would match the other dependencies.

Reply:
Done